src/Controls/samples/Controls.Sample.SingleProject/MainPage.xaml

```diff
@@ -8,6 +8,6 @@
           <RowDefinition Height="*" />
         </Grid.RowDefinitions>
         <Label Text="Hello, .NET MAUI Single Project!" TextColor="Black" FontSize="32" HorizontalOptions="CenterAndExpand" />
-        <WebView Grid.Row="1" Source="index.html" />
+        <WebView Grid.Row="1" Source="Resources/Raw/index.html" />
```

Comment:
So the file `<MauiAsset Include="Resources\Raw\index.html" />` now has to be accessed with the folder structure above.

Reply:
Hmm, iOS failed due to:
```
Resources/Raw/index.html The name 'Resources' is reserved and cannot be used. [Maui.Controls.Sample.SingleProject.csproj]
```
So this would have worked if `Resources` was another folder name...

@Redth are we going to keep using the `Resources` folder in templates? At what level should the directory structure be preserved?

Reply:
We can use Assets as well... Or even not use a resources folder and have it at the root like the HelloMaui sample.